File: OPENAPI-REST-API/swagger-client/micronaut/src/main/java/io/swagger/model/IdMigrationBody.java
package io.swagger.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.v3.oas.annotations.media.Schema;
import io.micronaut.core.annotation.Introspected;
import io.micronaut.validation.Validated;
import javax.validation.Valid;
import javax.validation.constraints.*;

/**
 * IdMigrationBody
 */
@Validated
@Introspected

public class IdMigrationBody   {
  @JsonProperty("custPortal")
  private String custPortal = null;

  @JsonProperty("regEmail")
  private String regEmail = null;

  @JsonProperty("password")
  private String password = null;

  @JsonProperty("ctrlPanel")
  private String ctrlPanel = null;

  @JsonProperty("ftpUsername")
  private String ftpUsername = null;

  @JsonProperty("ftpPassword")
  private String ftpPassword = null;

  @JsonProperty("siteBusyMig")
  private String siteBusyMig = null;

  @JsonProperty("splReqMig")
  private String splReqMig = null;

  @JsonProperty("domainReg")
  private String domainReg = null;

  @JsonProperty("dataMig")
  private String dataMig = null;

  @JsonProperty("domainRegPortal")
  private String domainRegPortal = null;

  @JsonProperty("domainRegEmail")
  private String domainRegEmail = null;

  @JsonProperty("domainRegPassword")
  private String domainRegPassword = null;

  public IdMigrationBody custPortal(String custPortal) {
    this.custPortal = custPortal;
    return this;
  }

  /**
   * URL of the customer's current hosting portal.
   * @return custPortal
  **/
  @Schema(description = "URL of the customer's current hosting portal.")
  @NotNull

  public String getCustPortal() {
    return custPortal;
  }

  public void setCustPortal(String custPortal) {
    this.custPortal = custPortal;
  }

  public IdMigrationBody regEmail(String regEmail) {
    this.regEmail = regEmail;
    return this;
  }

  /**
   * Registered email address at the current host.
   * @return regEmail
  **/
  @Schema(description = "Registered email address at the current host.")
  @NotNull

  public String getRegEmail() {
    return regEmail;
  }

  public void setRegEmail(String regEmail) {
    this.regEmail = regEmail;
  }

  public IdMigrationBody password(String password) {
    this.password = password;
    return this;
  }

  /**
   * Password for the current hosting account.
   * @return password
  **/
  @Schema(description = "Password for the current hosting account.")
  @NotNull

  public String getPassword() {
    return password;
  }

  public void setPassword(String password) {
    this.password = password;
  }

  public IdMigrationBody ctrlPanel(String ctrlPanel) {
    this.ctrlPanel = ctrlPanel;
    return this;
  }

  /**
   * URL of the current control panel.
   * @return ctrlPanel
  **/
  @Schema(description = "URL of the current control panel.")
  @NotNull

  public String getCtrlPanel() {
    return ctrlPanel;
  }

  public void setCtrlPanel(String ctrlPanel) {
    this.ctrlPanel = ctrlPanel;
  }

  public IdMigrationBody ftpUsername(String ftpUsername) {
    this.ftpUsername = ftpUsername;
    return this;
  }

  /**
   * FTP username at the current host.
   * @return ftpUsername
  **/
  @Schema(description = "FTP username at the current host.")
  @NotNull

  public String getFtpUsername() {
    return ftpUsername;
  }

  public void setFtpUsername(String ftpUsername) {
    this.ftpUsername = ftpUsername;
  }

  public IdMigrationBody ftpPassword(String ftpPassword) {
    this.ftpPassword = ftpPassword;
    return this;
  }

  /**
   * FTP password at the current host.
   * @return ftpPassword
  **/
  @Schema(description = "FTP password at the current host.")
  @NotNull

  public String getFtpPassword() {
    return ftpPassword;
  }

  public void setFtpPassword(String ftpPassword) {
    this.ftpPassword = ftpPassword;
  }

  public IdMigrationBody siteBusyMig(String siteBusyMig) {
    this.siteBusyMig = siteBusyMig;
    return this;
  }

  /**
   * Information about site traffic during migration.
   * @return siteBusyMig
  **/
  @Schema(description = "Information about site traffic during migration.")
  @NotNull

  public String getSiteBusyMig() {
    return siteBusyMig;
  }

  public void setSiteBusyMig(String siteBusyMig) {
    this.siteBusyMig = siteBusyMig;
  }

  public IdMigrationBody splReqMig(String splReqMig) {
    this.splReqMig = splReqMig;
    return this;
  }

  /**
   * Special requirements for the migration.
   * @return splReqMig
  **/
  @Schema(description = "Special requirements for the migration.")
  @NotNull

  public String getSplReqMig() {
    return splReqMig;
  }

  public void setSplReqMig(String splReqMig) {
    this.splReqMig = splReqMig;
  }

  public IdMigrationBody domainReg(String domainReg) {
    this.domainReg = domainReg;
    return this;
  }

  /**
   * Whether domain registration assistance is needed.
   * @return domainReg
  **/
  @Schema(description = "Whether domain registration assistance is needed.")
  @NotNull

  public String getDomainReg() {
    return domainReg;
  }

  public void setDomainReg(String domainReg) {
    this.domainReg = domainReg;
  }

  public IdMigrationBody dataMig(String dataMig) {
    this.dataMig = dataMig;
    return this;
  }

  /**
   * Data migration timing preference.
   * @return dataMig
  **/
  @Schema(description = "Data migration timing preference.")
  @NotNull

  public String getDataMig() {
    return dataMig;
  }

  public void setDataMig(String dataMig) {
    this.dataMig = dataMig;
  }

  public IdMigrationBody domainRegPortal(String domainRegPortal) {
    this.domainRegPortal = domainRegPortal;
    return this;
  }

  /**
   * Domain registrar portal URL.
   * @return domainRegPortal
  **/
  @Schema(description = "Domain registrar portal URL.")
  @NotNull

  public String getDomainRegPortal() {
    return domainRegPortal;
  }

  public void setDomainRegPortal(String domainRegPortal) {
    this.domainRegPortal = domainRegPortal;
  }

  public IdMigrationBody domainRegEmail(String domainRegEmail) {
    this.domainRegEmail = domainRegEmail;
    return this;
  }

  /**
   * Email for the domain registrar account.
   * @return domainRegEmail
  **/
  @Schema(description = "Email for the domain registrar account.")
  @NotNull

  public String getDomainRegEmail() {
    return domainRegEmail;
  }

  public void setDomainRegEmail(String domainRegEmail) {
    this.domainRegEmail = domainRegEmail;
  }

  public IdMigrationBody domainRegPassword(String domainRegPassword) {
    this.domainRegPassword = domainRegPassword;
    return this;
  }

  /**
   * Password for the domain registrar account.
   * @return domainRegPassword
  **/
  @Schema(description = "Password for the domain registrar account.")
  @NotNull

  public String getDomainRegPassword() {
    return domainRegPassword;
  }

  public void setDomainRegPassword(String domainRegPassword) {
    this.domainRegPassword = domainRegPassword;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    IdMigrationBody idMigrationBody = (IdMigrationBody) o;
    return Objects.equals(this.custPortal, idMigrationBody.custPortal) &&
        Objects.equals(this.regEmail, idMigrationBody.regEmail) &&
        Objects.equals(this.password, idMigrationBody.password) &&
        Objects.equals(this.ctrlPanel, idMigrationBody.ctrlPanel) &&
        Objects.equals(this.ftpUsername, idMigrationBody.ftpUsername) &&
        Objects.equals(this.ftpPassword, idMigrationBody.ftpPassword) &&
        Objects.equals(this.siteBusyMig, idMigrationBody.siteBusyMig) &&
        Objects.equals(this.splReqMig, idMigrationBody.splReqMig) &&
        Objects.equals(this.domainReg, idMigrationBody.domainReg) &&
        Objects.equals(this.dataMig, idMigrationBody.dataMig) &&
        Objects.equals(this.domainRegPortal, idMigrationBody.domainRegPortal) &&
        Objects.equals(this.domainRegEmail, idMigrationBody.domainRegEmail) &&
        Objects.equals(this.domainRegPassword, idMigrationBody.domainRegPassword);
  }

  @Override
  public int hashCode() {
    return Objects.hash(custPortal, regEmail, password, ctrlPanel, ftpUsername, ftpPassword, siteBusyMig, splReqMig, domainReg, dataMig, domainRegPortal, domainRegEmail, domainRegPassword);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class IdMigrationBody {\n");
    
    sb.append("    custPortal: ").append(toIndentedString(custPortal)).append("\n");
    sb.append("    regEmail: ").append(toIndentedString(regEmail)).append("\n");
    sb.append("    password: ").append(toIndentedString(password)).append("\n");
    sb.append("    ctrlPanel: ").append(toIndentedString(ctrlPanel)).append("\n");
    sb.append("    ftpUsername: ").append(toIndentedString(ftpUsername)).append("\n");
    sb.append("    ftpPassword: ").append(toIndentedString(ftpPassword)).append("\n");
    sb.append("    siteBusyMig: ").append(toIndentedString(siteBusyMig)).append("\n");
    sb.append("    splReqMig: ").append(toIndentedString(splReqMig)).append("\n");
    sb.append("    domainReg: ").append(toIndentedString(domainReg)).append("\n");
    sb.append("    dataMig: ").append(toIndentedString(dataMig)).append("\n");
    sb.append("    domainRegPortal: ").append(toIndentedString(domainRegPortal)).append("\n");
    sb.append("    domainRegEmail: ").append(toIndentedString(domainRegEmail)).append("\n");
    sb.append("    domainRegPassword: ").append(toIndentedString(domainRegPassword)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/IdMigrationBody.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class IdMigrationBody {
    /// <summary>
    /// URL of the customer's current hosting portal.
    /// </summary>
    /// <value>URL of the customer's current hosting portal.</value>
    [DataMember(Name="custPortal", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "custPortal")]
    public string CustPortal { get; set; }

    /// <summary>
    /// Registered email address at the current host.
    /// </summary>
    /// <value>Registered email address at the current host.</value>
    [DataMember(Name="regEmail", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "regEmail")]
    public string RegEmail { get; set; }

    /// <summary>
    /// Password for the current hosting account.
    /// </summary>
    /// <value>Password for the current hosting account.</value>
    [DataMember(Name="password", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "password")]
    public string Password { get; set; }

    /// <summary>
    /// URL of the current control panel.
    /// </summary>
    /// <value>URL of the current control panel.</value>
    [DataMember(Name="ctrlPanel", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "ctrlPanel")]
    public string CtrlPanel { get; set; }

    /// <summary>
    /// FTP username at the current host.
    /// </summary>
    /// <value>FTP username at the current host.</value>
    [DataMember(Name="ftpUsername", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "ftpUsername")]
    public string FtpUsername { get; set; }

    /// <summary>
    /// FTP password at the current host.
    /// </summary>
    /// <value>FTP password at the current host.</value>
    [DataMember(Name="ftpPassword", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "ftpPassword")]
    public string FtpPassword { get; set; }

    /// <summary>
    /// Information about site traffic during migration.
    /// </summary>
    /// <value>Information about site traffic during migration.</value>
    [DataMember(Name="siteBusyMig", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "siteBusyMig")]
    public string SiteBusyMig { get; set; }

    /// <summary>
    /// Special requirements for the migration.
    /// </summary>
    /// <value>Special requirements for the migration.</value>
    [DataMember(Name="splReqMig", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "splReqMig")]
    public string SplReqMig { get; set; }

    /// <summary>
    /// Whether domain registration assistance is needed.
    /// </summary>
    /// <value>Whether domain registration assistance is needed.</value>
    [DataMember(Name="domainReg", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "domainReg")]
    public string DomainReg { get; set; }

    /// <summary>
    /// Data migration timing preference.
    /// </summary>
    /// <value>Data migration timing preference.</value>
    [DataMember(Name="dataMig", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "dataMig")]
    public string DataMig { get; set; }

    /// <summary>
    /// Domain registrar portal URL.
    /// </summary>
    /// <value>Domain registrar portal URL.</value>
    [DataMember(Name="domainRegPortal", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "domainRegPortal")]
    public string DomainRegPortal { get; set; }

    /// <summary>
    /// Email for the domain registrar account.
    /// </summary>
    /// <value>Email for the domain registrar account.</value>
    [DataMember(Name="domainRegEmail", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "domainRegEmail")]
    public string DomainRegEmail { get; set; }

    /// <summary>
    /// Password for the domain registrar account.
    /// </summary>
    /// <value>Password for the domain registrar account.</value>
    [DataMember(Name="domainRegPassword", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "domainRegPassword")]
    public string DomainRegPassword { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class IdMigrationBody {\n");
      sb.Append("  CustPortal: ").Append(CustPortal).Append("\n");
      sb.Append("  RegEmail: ").Append(RegEmail).Append("\n");
      sb.Append("  Password: ").Append(Password).Append("\n");
      sb.Append("  CtrlPanel: ").Append(CtrlPanel).Append("\n");
      sb.Append("  FtpUsername: ").Append(FtpUsername).Append("\n");
      sb.Append("  FtpPassword: ").Append(FtpPassword).Append("\n");
      sb.Append("  SiteBusyMig: ").Append(SiteBusyMig).Append("\n");
      sb.Append("  SplReqMig: ").Append(SplReqMig).Append("\n");
      sb.Append("  DomainReg: ").Append(DomainReg).Append("\n");
      sb.Append("  DataMig: ").Append(DataMig).Append("\n");
      sb.Append("  DomainRegPortal: ").Append(DomainRegPortal).Append("\n");
      sb.Append("  DomainRegEmail: ").Append(DomainRegEmail).Append("\n");
      sb.Append("  DomainRegPassword: ").Append(DomainRegPassword).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
